packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec

```diff
@@ -17,7 +17,7 @@
 
 Summary:    MaaS Discovery Plugin for Foreman
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    %{mainver}
+Version:    16.1.2
```

Comment:
As in the `develop` PR, please use the macro, or kill it (for a stable branch, I'd prefer a smaller diff)

Reply:
Removed.